docs/integrations/sources/mailchimp.md

```diff
@@ -1,65 +1,260 @@
-| `int`, `float`, `number` | `number` |  |
-| `object` | `object` | properties within objects are mapped based on the mappings in this table |
-| `string` | `string` |  |
+[Log in](https://login.mailchimp.com/) to Mailchimp account. 
+If you don't have a Mailchimp account already, you’ll need to [create](https://login.mailchimp.com/signup/) one in order to use the API. 
 
-### Features
+## Step 2: Set up the source connector in Airbyte
 
-| Feature | Supported?\(Yes/No\) | Notes |
-| :--- | :--- | :--- |
-| Full Refresh Sync | Yes |  |
-| Incremental Sync | Coming soon |  |
-| Replicate Incremental Deletes | Coming soon |  |
-| SSL connection | Yes | Enabled by default |
-| Namespaces | No |  |
+**For Airbyte Cloud:**
 
-### Performance considerations
+1. For using [OAuth2.0](https://mailchimp.com/developer/marketing/guides/access-user-data-oauth-2/) creds, 
+please [register](https://mailchimp.com/developer/marketing/guides/access-user-data-oauth-2/#register-your-application) 
+your Mailchimp account.
+2. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+3. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**.
+4. On the Set up the source page, enter the name for the Mailchimp connector and select **Mailchimp** from the Source type dropdown.
+5. Select `OAuth2.0` Authorization method, then click `Authenticate your account`.
+6. Log in and Authorize to the Mailchimp account and click `Set up source`.
 
-At the time of this writing, [Mailchimp does not impose rate limits](https://mailchimp.com/developer/guides/marketing-api-conventions/#throttling) on how much data is read form its API in a single sync process. However, Mailchimp enforces a maximum of 10 simultaneous connections to its API. This means that Airbyte will not be able to run more than 10 concurrent syncs from Mailchimp using API keys generated from the same account.
+**For Airbyte OSS:**
 
-## Getting started
+1. For using an API key [create account](https://mailchimp.com/developer/marketing/guides/quick-start/#create-an-account) 
+in Mailchimp.
+2. [Generate](https://mailchimp.com/developer/marketing/guides/quick-start/#generate-your-api-key) API key. 
+3. Go to local Airbyte page.
+4. In the left navigation bar, click **Sources**. In the top-right corner, click **+ new source**. 
+5. On the Set up the source page, enter the name for the Mailchimp connector and select **Mailchimp** from the Source type dropdown. 
+6. Select `API key` Authorization method, then copy and paste your API key from step 2. 
+7. Click `Set up source`.
 
-### Requirements
 
-For Apikey authorithation:
-* Mailchimp account 
-* Mailchimp API key
+## Supported sync modes
 
-For OAuth authorization:
-* Mailchimp registered app
-* Mailchimp client_id
-* Mailchimp client_secret
+The Mailchimp source connector supports the following [sync modes](https://docs.airbyte.com/getting-started-with-airbyte-cloud/core-concepts#connection-sync-modes):
```

Comment:
Looks like this link is broken?

cc: @Amruta-Ranade are all of these links broken due to the migration?